src/options.js

```diff
@@ -39,15 +41,18 @@ export function getRealmOptions({
   debugNames = false,
   uniqueSuffix,
   timeout,
+  residual,
+  serialize,
```

Comment:
getRealmOptions should allow its caller to specify the values of residual and serialize. I think what you want is for defaultOptions to set serialize to true.